doc/source/_toc.yml

```diff
@@ -256,27 +256,33 @@ parts:
         sections:
           - file: serve/getting_started
           - file: serve/key-concepts
-          - file: serve/user-guide
+          - file: serve/model_composition
+          - file: serve/deploy-many-models/index
             sections:
-              - file: serve/http-guide
-              - file: serve/scaling-and-resource-allocation
-              - file: serve/model_composition
-              - file: serve/dev-workflow
-              - file: serve/app-builder-guide
-              - file: serve/multi-app
-              - file: serve/production-guide/index
-                sections:
-                  - file: serve/production-guide/config
-                  - file: serve/production-guide/deploy-vm
-                  - file: serve/production-guide/kubernetes
-                  - file: serve/production-guide/monitoring
-                  - file: serve/production-guide/fault-tolerance
-              - file: serve/performance
-              - file: serve/handling-dependencies
-              - file: serve/managing-java-deployments
-              - file: serve/migration
-              - file: serve/direct-ingress
-              - file: serve/model-multiplexing
+            - file: serve/deploy-many-models/multi-app
+            - file: serve/deploy-many-models/model-multiplexing
+          - file: serve/production-guide/index
+            sections:
+              - file: serve/production-guide/config
+              - file: serve/production-guide/kubernetes
+              - file: serve/production-guide/handling-dependencies
+              - file: serve/production-guide/best-practices
+          - file: serve/monitoring
+          - file: serve/scaling-and-resource-allocation
+          - file: serve/advanced-guides/index
+            sections:
+              - file: serve/advanced-guides/app-builder-guide
+              - file: serve/advanced-guides/http-guide
```

Comment:
I don't think this one should be advanced, it has some pretty basic info like how to use FastAPI integration